src/shared/Typography/styles.ts

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components';
+import Colors from '../colors';
+
+export const H1 = styled.h1`
+  font-family: 'Montserrat', sans-serif;
```

Comment:
I think you can create a variable for the font so as not to duplicate it

Reply:
done